### ISSUE TITLE
Accessibility options tool tip is not appearing on editor content  [ MAS 1.3.3 ]  

### ISSUE THREAD

**Issue by rebro-msft** [microsoft/monaco-editor#1320]
**monaco-editor version:** 0.14.3
**Browser:** All
**OS:** N/A

**Issue:**
When the user is using a screen reader and tabs into the editor area, it reads "Press Alt+F1 for accessibility options" because of the aria-label.  But if the user is not using a screen reader and is simply tabbing via the keyboard, they are not aware that these accessibility options exist. 

**Suggested solution:** 
Enable a tooltip for the editor area that displays the accessibility options message.

**Comment by alexandrudima** [comment]
I think we are doing enough given we are a widget. I think that the integrator can do more on their side.

For example, perhaps in the website documentation or in some other central location, accessibility features of an application can be documented. The Monaco Editor is used in a variety of use-cases, sometimes tens or hundreds of editors are instantiated per page. Having each and every one show some sort of tooltip to promote the high contrast theme or the tab moves focus toggling command is, IMHO too much of a requirement from a widget.

TL;DR I think the integrator can do more to promote and offer help / teach about accessibility options, as indicated here -- https://github.com/Microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators

In what application are you using the Monaco Editor ?

**Comment by smhigley** [comment]
@alexandrudima thanks for the response! I agree that it makes sense for the integrator to control whether or not to show a tooltip based on user settings/previous tooltip dismissal/whatever else.

Would you be at all open to adding a boolean property to the `IEditorConstructionOptions` like `showF1Tooltip` or `showAriaLabelTooltip` that defaults to false, but can be set to true by the integrator? That would allow the tooltip (if true) to be triggered by focus on the textarea, which would benefit sighted screen reader users more than having a tooltip completely separated from the `ariaLabel`.

Let me know what you think, and if that approach sounds good to you, let me know if there's anything I can do to help with the tooltip's accessibility.

**Comment by alexandrudima** [comment]
@smhigley I think help messages can be easily presented outside of the editor core. It is also possible to use editor's API such as listening to the focus event to highlight help messages if this is what you want to do...
